test/specs/marked/marked-spec.js

```diff
@@ -32,6 +32,21 @@ Messenger.prototype.test = function(spec, section, ignore) {
 
 var messenger = new Messenger();
 
+describe('Marked Autolinks', function() {
+  var section = 'Autolinks';
+
+  // var shouldPassButFails = [];
```

Comment:
Not sure why this line is here?

Reply:
It's a copy/paste from elsewhere in the same file. See lines 53 and 68. I'll remove them all.